myriad-scheduler/src/test/java/org/apache/myriad/scheduler/TestMyriadScheduler.java

```diff
@@ -22,17 +22,18 @@
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.yarn.conf.YarnConfiguration;
 import org.apache.hadoop.yarn.event.AsyncDispatcher;
-//import org.apache.hadoop.yarn.server.resourcemanager.MockNodes;
 import org.apache.hadoop.yarn.server.resourcemanager.ResourceManager;
-//import org.apache.hadoop.yarn.server.resourcemanager.rmnode.RMNode;
 import org.apache.hadoop.yarn.server.resourcemanager.scheduler.ResourceScheduler;
-//import org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.NodeAddedSchedulerEvent;
-//import org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.NodeRemovedSchedulerEvent;
 import org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler;
 import org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairSchedulerConfiguration;
-//import org.apache.hadoop.yarn.util.resource.Resources;
 import org.junit.After;
 import org.junit.Before;
+
+//import org.apache.hadoop.yarn.server.resourcemanager.MockNodes;
+//import org.apache.hadoop.yarn.server.resourcemanager.rmnode.RMNode;
+//import org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.NodeAddedSchedulerEvent;
+//import org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.NodeRemovedSchedulerEvent;
+//import org.apache.hadoop.yarn.util.resource.Resources;
 //import org.junit.Test;
 
```

Comment:
this should probably be part of a bigger code cleanup. Keeping commented
code lying around is just bad, that's what git is for.

On Thu, Oct 29, 2015 at 11:31 AM, Darin <notifications@github.com> wrote:

> In
> myriad-scheduler/src/test/java/org/apache/myriad/scheduler/TestMyriadScheduler.java
> <https://github.com/apache/incubator-myriad/pull/26#discussion_r43400247>:
>
> >  import org.junit.After;
> >  import org.junit.Before;
> > +
> > +//import org.apache.hadoop.yarn.server.resourcemanager.MockNodes;
> > +//import org.apache.hadoop.yarn.server.resourcemanager.rmnode.RMNode;
> > +//import org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.NodeAddedSchedulerEvent;
> > +//import org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.NodeRemovedSchedulerEvent;
> > +//import org.apache.hadoop.yarn.util.resource.Resources;
> >  //import org.junit.Test;
> >
>
> Do we really want these commented imports? If unnecessary delete.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-myriad/pull/26/files#r43400247>.
>